Examples/Python/python/acts/examples/simulation.py

```diff
@@ -625,16 +627,27 @@ def addGeant4(
 
     # Selector
     if postSelectParticles is not None:
-        postSelectedParticles = "particles_initial_selected"
+        particlesInitial = "particles_initial_selected"
         addParticleSelection(
             s,
-            preSelectParticles,
+            postSelectParticles,
             inputParticles=g4conf.outputParticlesInitial,
-            outputParticles=postSelectedParticles,
+            outputParticles=particlesInitial,
+        )
+
+        particlesFinal = "particles_final_selected"
+        addParticleSelection(
+            s,
+            postSelectParticles,
+            inputParticles=g4conf.outputParticlesFinal,
+            outputParticles=particlesFinal,
         )
-        s.addWhiteboardAlias("particles", postSelectedParticles)
     else:
-        s.addWhiteboardAlias("particles", alg.config.outputParticlesInitial)
+        particlesInitial = alg.config.outputParticlesInitial
+        particlesFinal = alg.config.outputParticlesFinal
+
+    # Only add alieas for 'particles_initial' as this is the one we use most
+    s.addWhiteboardAlias("particles", particlesInitial)
```

Comment:
I think we should reflect all these changes in `addFatras` too

Reply:
yeah good point, I'll update it later today

Reply:
Was this done?